### PR TITLE
PMM-10549 local backup

### PR DESCRIPTION
**Pull request by fabio-silva**
**What this PR does / why we need it**: Enables backing up to local client.

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-10549

https://github.com/Percona-Lab/pmm-submodules/pull/2742

### CODE REVIEW

File: public/app/percona/shared/components/Elements/SecretToggler/SecretToggler.styles.ts [hunk 1]
```diff
@@ -14,6 +14,6 @@ export const getStyles = ({ spacing }: GrafanaTheme) => ({
   fullLock: css`
     position: absolute;
     right: 7px;
-    top: 30px;
+    top: 34px;
```
Comment (solovevayaroslavna) [inline public/app/percona/shared/components/Elements/SecretToggler/SecretToggler.styles.ts:17]: why we need this magic number? Don't we have any suitable in grafana theme?

**Review by solovevayaroslavna — DISMISSED** [review]

File: public/app/percona/shared/components/Elements/SecretToggler/SecretToggler.styles.ts (hunk 1 above)
Comment (fabio-silva) [reply public/app/percona/shared/components/Elements/SecretToggler/SecretToggler.styles.ts:17]: Hm, not really. This is tied to the height of our own TextInputField, so I had to manually position it at the center using this :(

**Review by filipmikes1 — APPROVED** [review]

File: public/app/percona/backup/components/BackupInventory/BackupInventory.tsx [hunk 2]
```diff
@@ -304,8 +327,15 @@ export const BackupInventory: FC = () => {
               onDelete={handleDelete}
               initialForceValue={true}
               loading={deletePending}
-              showForce
-            />
+              showForce={!!selectedBackup && backupLocationMap.current[selectedBackup.id]?.type !== LocationType.CLIENT}
+            >
+              {!!selectedBackup && backupLocationMap.current[selectedBackup.id]?.type === LocationType.CLIENT && (
+                <Alert title="">
+                  For security reasons, we won&apos;t delete this backup from the filesystem. Please make sure to
```
Comment (filipmikes1) [inline public/app/percona/backup/components/BackupInventory/BackupInventory.tsx:334]: Can you move this text to a separate file?

Comment (fabio-silva) [reply public/app/percona/backup/components/BackupInventory/BackupInventory.tsx]: sure!

**Comment by fabio-silva** [comment]
@solovevayaroslavna @filipmikes1 please re-review, QA accepted it

**Review by solovevayaroslavna — APPROVED** [review]

**Pull request merged**